nodeup/pkg/model/kubelet.go

```diff
@@ -146,6 +148,17 @@ func (b *KubeletBuilder) buildSystemdEnvironmentFile(kubeletConfig *kops.Kubelet
 	if b.UsesCNI() {
 		flags += " --cni-bin-dir=" + b.CNIBinDir()
 		flags += " --cni-conf-dir=" + b.CNIConfDir()
+		// If we are using the AmazonVPC plugin we need to bind the kubelet to the local ipv4 address
+		if b.Cluster.Spec.Networking.AmazonVPC != nil {
+			sess := session.Must(session.NewSession())
+			metadata := ec2metadata.New(sess)
+			localIpv4, err := metadata.GetMetadata("local-ipv4")
+			if err != nil {
+				glog.Fatalf("Couldn't fetch the local-ipv4 address from the ec2 meta-data: %v", err)
+			} else {
+				flags += " --node-ip=" + localIpv4
```

Comment:
Don’t need the else as I think glog.Fatal will return an err. We may want to log and error and return err here.